src/methods/wallet-get-capabilities.ts

```diff
@@ -0,0 +1,45 @@
+import { rpcErrors } from '@metamask/rpc-errors';
+import type { Infer } from '@metamask/superstruct';
+import { tuple } from '@metamask/superstruct';
+import type {
+  Hex,
+  Json,
+  JsonRpcRequest,
+  PendingJsonRpcResponse,
+} from '@metamask/utils';
+import { HexChecksumAddressStruct } from '@metamask/utils';
+
+import { validateParams } from '../utils/validation';
+
+const GetCapabilitiesStruct = tuple([HexChecksumAddressStruct]);
+
+export type GetCapabilitiesParams = Infer<typeof GetCapabilitiesStruct>;
+export type GetCapabilitiesResult = Record<Hex, Record<string, any>>;
```

Comment:
i know the specs say this is `any`, but is it actually `any`?

Reply:
Fair point, have changed to `Json` since it's being sent over JSON-RPC so is technically limited.